src/applications/mhv-medical-records/containers/DownloadReportPage.jsx

```diff
@@ -407,6 +428,9 @@ const DownloadReportPage = ({ runningUnitTest }) => {
           <p className="vads-u-margin--0">
             This report includes all the health information you entered yourself
             in the previous version of My HealtheVet.
+            {selectMilestoneTwoFlag &&
+              ` You can no longer enter or
+            edit health information in My HealtheVet.`}
```

Comment:
I see your screenshot, but I am not seeing the text from the Figma designs in the code.
![image](https://github.com/user-attachments/assets/4b274d1b-fd3e-4ee2-bab8-f801adaf19af)